docs/docs/cmd/pa/app/app-permission-ensure.mdx

```diff
@@ -0,0 +1,85 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# pa app permission ensure
+
+Assigns/updates permissions to a Power Apps app
+
+## Usage
+
+```sh
+m365 pa app permission ensure [options]
+```
+
+## Options
+
+```md definition-list
+`--appName <appName>`
+: The name (GUID) of the Microsoft Power App.
+
+`--roleName <roleName>`
+: Permission level given to the app. Valid values: `CanEdit`, `CanView`. Sharing with the entire tenant is only supported with `CanView` role.
+
+`--userId [userId]`
+: The ID of the user. Specify either `userId`, `userName`, `groupId`, `groupName`, or `tenant`.
+
+`--userName [userName]`
+: User principal name of the user. Specify either `userId`, `userName`, `groupId`, `groupName`, or `tenant`.
+
+`--groupId [groupId]`
+: The ID of the group. Specify either `userId`, `userName`, `groupId`, `groupName`, or `tenant`.
+
+`--groupName [groupName]`
+: The name of the group. Specify either `userId`, `userName`, `groupId`, `groupName`, or `tenant`.
+
+`--tenant`
+: Share the app with the entire tenant. When specifying this, only `CanView` role is supported. Specify either `userId`, `userName`, `groupId`, `groupName`, or `tenant`.
+
+`-e, --environmentName [environmentName]`
+: The name of the environment. Specify only a value when running the command as admin.
+
+`--sendInvitationMail`
```

Comment:
Let's move this option up so the options to run this command as admin are grouped together.

- sendInvitationMail
- asAdmin
- environmentName